modules/configuring-a-floating-action-button/proc-configuring-floating-action-button-as-a-dynamic-plugin.adoc

```diff
@@ -0,0 +1,149 @@
+
+You can configure the floating action button as a dynamic plugin to perform actions or open an internal or external link.
+
+.Prerequisties
+You must have sufficient permissions as a platform engineer.
+
+.Procedure
+
+To configure a floating action button as a dynamic plugin, complete any of the following tasks:
+
+* Specify the `global.floatingactionbutton/config` mount point in your `app-config-dynamic.yaml` file. For example: 
++
+.Example of a bulk-import plugin as a floating action button
+[source,yaml]
+----
+  dynamicPlugins:
+    frontend:
+      red-hat-developer-hub.backstage-plugin-bulk-import:
+        # Start of the floating action button configuration
+        mountPoints:
+          - mountPoint: global.floatingactionbutton/config
+            importName: BulkImportPage <1>
+            config:
+              slot: 'page-end'
+              icon: bulkImportIcon
+              label: 'Bulk import'
+              toolTip: 'Register multiple repositories in bulk'
+              to: /bulk-import/repositories
+        # End of the floating action button configuration
+        appIcons:
+          - name: bulkImportIcon
+            importName: BulkImportIcon
+        dynamicRoutes:
+          - path: /bulk-import/repositories
+            importName: BulkImportPage
+            menuItem:
+              icon: bulkImportIcon
+              text: Bulk import
+----
+<1> (Required) The import name with an associated component to the mount point.
+
+* To configure a dynamic plugin as a floating action button that opens an external link, specify the `global.floatingactionbutton/config` mount point in your `app-config-dynamic.yaml` file. For example:
++
+.Example of a floating action button that opens GitHub
+[source,yaml]
+----
+  dynamicPlugins:
+    frontend:
+      red-hat-developer-hub.backstage-plugin-global-floating-action-button:
+        mountPoints:
+          - mountPoint: application/listener
+            importName: DynamicGlobalFloatingActionButton
+          - mountPoint: global.floatingactionbutton/config
+            importName: NullComponent
+            config:
+              icon: github
+              label: 'Git'
+              toolTip: 'Github'
+             	to: https://github.com/redhat-developer/rhdh-plugins
+----
+
+== Floating action button parameters
+Use the parameters as shown in the following table to configure your floating action button plugin.
+
+.Floating action button parameters
+|===
+| Name | Description | Type | Default value | Required
+
+| `slot`
+| Position of the floating action button. Valid values: `PAGE_END`, `BOTTOM_LEFT`
+| `enum`
+| `PAGE_END`
+| No
+
+| `label`
+| Name of the floating action button
+| `String`
+| Not applicable
+| Yes
+
+| `icon`
+| Icon of the floating action button. Recommended to use filled icons from the link:https://fonts.google.com/icons[Material Design library].
+| `StringReact.ReactElement`
+| Not applicable
```

Comment:
even svg icon and html image is applicable
I have an example here with an SVG icon : https://github.com/redhat-developer/rhdh-plugins/tree/main/workspaces/global-floating-action-button/plugins/global-floating-action-button#installing-as-a-dynamic-plugin

I will update the frontend plugin readme to include this